packages/neo-one-monitor/src/types.js

```diff
@@ -197,6 +198,24 @@ export interface Summary {
   observe(labels: Labels, value: number): void;
 }
 
+export interface CounterMetric {
```

Comment:
These should stay in `MonitorBase` - they don't have anything to do with the public api of `Monitor` (which is what's in this file).